packages/experimental/RadioGroup/src/Radio/useRadio.ts

```diff
@@ -9,10 +9,15 @@ const defaultAccessibilityActions = [{ name: 'Select' }];
 
 export const useRadio = (props: RadioProps): RadioState => {
   const defaultComponentRef = React.useRef(null);
+
+  // Grabs the context information from RadioGroup (currently selected button and client's onChange callback)
+  const selectedInfo = useRadioGroupContext();
```

Comment:
nit: Can the name of this be something like "context" or "radioGroupContext" instead? "selectedInfo" doesn't seem like a good name for this since it has more than information about which radio is selected.

Reply:
I agree, it's not descriptive. Changed, thanks!